src/aeb_fate_generate_ops.erl

```diff
@@ -42,145 +42,145 @@ check_numbering(_, []) -> true.
+    , { 'BYTES_TO_STR',        16#77,  false,    true,   true,  3, [a,a],        bytes_to_str,                           {bytes},  string, "Arg0 := bytes_to_str(Arg1)"}
 
-    , { 'ORACLE_CHECK',        16#78,  false,   false,   3, [a,a,a,a],    oracle_check,        {oracle, typerep, typerep},    bool, "Arg0 := is Arg1 an oracle with the given query (Arg2) and response (Arg3) types"}
-    , { 'ORACLE_CHECK_QUERY',  16#79,  false,   false,   3, [a,a,a,a,a], oracle_check_query, {oracle, oracle_query, typerep, typerep}, bool, "Arg0 := is Arg2 a query for the oracle Arg1 with the given types (Arg3, Arg4)"}
+    , { 'ORACLE_CHECK',        16#78,  false,   false,   true,  3, [a,a,a,a],    oracle_check,        {oracle, typerep, typerep},    bool, "Arg0 := is Arg1 an oracle with the given query (Arg2) and response (Arg3) types"}
+    , { 'ORACLE_CHECK_QUERY',  16#79,  false,   false,   true,  3, [a,a,a,a,a], oracle_check_query, {oracle, oracle_query, typerep, typerep}, bool, "Arg0 := is Arg2 a query for the oracle Arg1 with the given types (Arg3, Arg4)"}
 
-    , { 'IS_ORACLE',           16#7a,  false,   false,   3, [a,a],         is_oracle,                           {address},    bool, "Arg0 := is Arg1 an oracle"}
-    , { 'IS_CONTRACT',         16#7b,  false,   false,   3, [a,a],       is_contract,                           {address},    bool, "Arg0 := is Arg1 a contract"}
-    , { 'CREATOR',             16#7c,  false,    true,   3, [a],    contract_creator,                                  {}, address, "Arg0 := contract creator"}
+    , { 'IS_ORACLE',           16#7a,  false,   false,   true,  3, [a,a],         is_oracle,                           {address},    bool, "Arg0 := is Arg1 an oracle"}
+    , { 'IS_CONTRACT',         16#7b,  false,   false,   true,  3, [a,a],       is_contract,                           {address},    bool, "Arg0 := is Arg1 a contract"}
+    , { 'CREATOR',             16#7c,  false,    true,   true,  3, [a],    contract_creator,                                  {}, address, "Arg0 := contract creator"}
 
-    , { 'DEACTIVATE',          16#fa,  false,    true,   3, [],           deactivate,                                  {},    none, "Mark the current contract for deactivation."}
-    , { 'ABORT',               16#fb,   true,    true,   3, [a],               abort,                            {string},    none, "Abort execution (dont use all gas) with error message in Arg0."}
-    , { 'EXIT',                16#fc,   true,    true,   3, [a],                exit,                            {string},    none, "Abort execution (use upp all gas) with error message in Arg0."}
-    , { 'NOP',                 16#fd,  false,    true,   1, [],                  nop,                                  {},    none, "The no op. does nothing."}
+    , { 'DEACTIVATE',          16#fa,  false,    true,   true,  3, [],           deactivate,                                  {},    none, "Mark the current contract for deactivation."}
```

Comment:
This is not clear to me.

Reply:
This operation is still not implemented. But I expect it to be used offchain as well to remove contracts in the offchain trees.